### PR TITLE
D-012 notifications

### PR DESCRIPTION
**Pull request by Piki2000**

### CODE REVIEW

File: pairing.md [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
-| Dev 1       | X           | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       |
-| Dev 2       | HH:MM       | X           | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       |
-| Dev 3       | HH:MM       | HH:MM       | X           | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       |
-| Dev 4       | HH:MM       | HH:MM       | HH:MM       | X           | HH:MM       | HH:MM       | HH:MM       | HH:MM       |
-| Dev 5       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | X           | HH:MM       | HH:MM       | HH:MM       |
-| Dev 6       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | X           | HH:MM       | HH:MM       |
-| Dev 7       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | X           | HH:MM       |
-| Dev 8       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | HH:MM       | X           |
+| Dev 1       | X           | HH:MM       | 17:00       | 07:30       | HH:MM       | 17:00       | 14:00       | HH:MM       |
```
Comment (ghinta) [inline pairing.md:13]: did you merge from the master or dev branch?

**Review by ghinta — COMMENTED** [review]
code looks clean, rdy to merge

File: pairing.md (hunk 1 above)
Comment (Piki2000) [reply pairing.md:13]: dev branch

Comment (Piki2000) [reply pairing.md:13]: i merged from the dev branch

**Pull request merged**